README.md

```diff
@@ -4,7 +4,7 @@
 
 ## What's blueprint?
 
-Blueprint is a light framework that replaces Nix glue code with a regular folder structure. Focus on deploying your infrastructure / package sets instead of reinventing the wheel.
+Blueprint replaces Nix glue code by a regular folder structure. Focus on deploying your infrastructure / package sets instead of reinventing the wheel.
```

Comment:
```suggestion
Blueprint replaces Nix glue code with a regular folder structure. Focus on deploying your infrastructure / package sets instead of reinventing the wheel.
```